controller/metrics/metrics.go

```diff
@@ -386,7 +386,7 @@ func (c *appCollector) collectApps(ch chan<- prometheus.Metric, app *argoappv1.A
 
 	autoSyncEnabled := app.Spec.SyncPolicy != nil && app.Spec.SyncPolicy.Automated != nil
 
-	addGauge(descAppInfo, 1, strconv.FormatBool(autoSyncEnabled), git.NormalizeGitURL(app.Spec.GetSource().RepoURL), app.Spec.Destination.Server, app.Spec.Destination.Namespace, string(syncStatus), string(healthStatus), operation)
+	addGauge(descAppInfo, 1, strconv.FormatBool(autoSyncEnabled), git.NormalizeGitURL(app.Spec.GetSource().RepoURL), app.Spec.Destination.Server, app.Spec.Destination.Namespace, string(syncStatus), string(healthStatus), operation, app.Spec.Source.TargetRevision)
```

Comment:
I think adding `spec.Source.TargetRevision` would not represent the correct revision of the application. It might be good to take the revision from the status of the app as it would rightly show the correct status the app is synced to.

Reply:
Yes I agree, I was considering what to choose between the two. Ill set the other one

Reply:
Done